src/renderer/components/opds/BrowserResult.tsx

```diff
@@ -114,15 +124,17 @@ export class BrowserResult extends React.Component<BrowserResultProps, IState> {
         </div>;
     }
 
-    private browseOpds() {
-        const { url, location } = this.props;
+    private browseOpds(url: string) {
+        this.setState({browserResult: undefined});
+        const { location } = this.props;
         const { browserResult } = this.state;
         const oldQs = parseQueryString(url.split("?")[1]);
         const search = qs.parse(location.search.replace("?", "")).search;
         let newUrl = url;
-
-        if (search && browserResult && browserResult.isSuccess && browserResult.data.searchUrl) {
-            newUrl = browserResult.data.searchUrl;
+        if (search && browserResult && typeof browserResult === "object"
```

Comment:
i think that is a bad merge conflict : `typeof browserResult === "object"` has been remove it

Reply:
This comment has been addressed: https://github.com/readium/readium-desktop/pull/733/commits/2fcf478637b8b6d3ef2462ae4a5739c36cad5aa7